tests/DatabaseTest.php

```diff
@@ -52,12 +54,12 @@ public function testUpdateUser()
         $user = $this->_createUser([
             'db-id' => $database->id,
             'login' => 'test_user1',
-            'password' => 'setup1Q',
+            'password' => PasswordProvider::getPassword(),
         ]);
         $updatedUser = static::$_client->database()->updateUser([
             'id' => $user->id,
             'login' => 'test_user2',
-            'password' => 'setup2Q',
+            'password' => PasswordProvider::getPassword() . '1',
```

Comment:
I think we can set default password for this user. No idea why we need change it for this user.

Reply:
Updated accordingly to the suggestion

Reply:
Okay, i'll check it tomorrow. Other things looks good now.